### PR TITLE
fix(regression): BottomNavigation can't render a single tab

### PR DESCRIPTION
**Pull request by brunohkbx**
### Summary

After rebasing #3408 I noticed that this regression has been introduced in V5. The BottomNavigation breaks when you try to render it with a single tab.

Issue

### Test plan

1. Open the example APP
2. Go to the example/src/Examples/BottomNavigationExample.tsx
4. Modify the routes to have a single one instead
5. Should render a single tab as in the following screenshot

Fixed

### CODE REVIEW

File: src/components/__tests__/BottomNavigation.test.js [hunk 1]
```diff
@@ -337,3 +337,17 @@ it('renders custom background color passed to barStyle property', () => {
...
+    <BottomNavigation
+      shifting={false}
+      navigationState={createState(0, 1)}
+      onIndexChange={jest.fn()}
+      renderScene={({ route }) => route.title}
+      testID={'bottom-navigation'}
+    />
+  );
+
+  getByTestId('bottom-navigation')
```
Comment (lukewalczak) [inline src/components/__tests__/BottomNavigation.test.js:352]: That test doesn't have any assertions

**Comment by lukewalczak** [comment]
@brunohkbx thanks for the fix, totally overlooked it. please only adjust the test and it will be ready to be shipped!

**Comment by brunohkbx** [comment]
@lukewalczak fixed ✅

**Review by lukewalczak — APPROVED** [review]
Thanks for the fix @brunohkbx!

**Pull request merged**